templates/common/fs.go

```diff
@@ -157,6 +157,8 @@ func CopyRecursive(ctx context.Context, pos *model.ConfigPos, p *CopyParams) (ou
 		if err != nil {
 			return err // There was some filesystem error. Give up.
 		}
+		logger.DebugContext(ctx, "CopyRecursive handling directory entry",
```

Comment:
Nit: there's already a "logger" field (L152), so this is duplicating "CopyRecursive":

```json
{"logger":"CopyRecursive", "message":"CopyRecursive handling directory entry"}
```

Reply:
Aha, thanks.